Content.Shared/Inventory/SlotFlags.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Inventory;
+
+/// <summary>
+///     Defines what slot types an item can fit into.
+/// </summary>
+[Serializable, NetSerializable]
+[Flags]
+public enum SlotFlags
+{
+    NONE = 0,
+    PREVENTEQUIP = 1 << 0,
+    HEAD = 1 << 1,
+    HELMET = 1 << 1,
+    EYES = 1 << 2,
+    EARS = 1 << 3,
+    MASK = 1 << 4,
+    OUTERCLOTHING = 1 << 5,
+    INNERCLOTHING = 1 << 6,
+    NECK = 1 << 7,
+    BACK = 1 << 8,
+    BACKPACK = 1 << 8,
+    BELT = 1 << 9,
+    GLOVES = 1 << 10,
+    HAND = 1 << 10,
+    IDCARD = 1 << 11,
+    POCKET = 1 << 12,
+    LEGS = 1 << 13,
+    SHOES = 1 << 14,
+    FEET = 1 << 14,
```

Comment:
How much work would it be to get rid of the duplicate values?
Cause I think that's causing VV to crash when looking at item components

Reply:
uhhhh. i copied this over and totally missed the duplicate value. im gonna remove the feet flag. its unused anyways

Reply:
FIY there's also head/helmet, hand/gloves, and back/backpack.
I think some of those aren't unused. Or do you just want to fix that in a future PR?